spec/Classes.tex

```diff
@@ -739,37 +1063,348 @@ \subsection{The Compiler-Generated Constructor}
 }
 \end{chapel}
 \begin{chapelpost}
-var c1 = new unmanaged C();
-var c2 = new unmanaged C(2);
-var c3 = new unmanaged C(z="");
-var c4 = new unmanaged C(2, z="");
-var c5 = new unmanaged C(0, 0.0, "");
+var c1 = new C();
+var c2 = new C(2);
+var c3 = new C(z="");
+var c4 = new C(2, z="");
+var c5 = new C(0, 0.0, "");
 writeln((c1, c2, c3, c4, c5));
-delete c1;
-delete c2;
-delete c3;
-delete c4;
-delete c5;
 \end{chapelpost}
+\begin{chapelcompopts}
+--no-warnings
+\end{chapelcompopts}
 \begin{chapeloutput}
 ({x = 0, y = 3.14, z = Hello, World!}, {x = 2, y = 3.14, z = Hello, World!}, {x = 0, y = 3.14, z = }, {x = 2, y = 3.14, z = }, {x = 0, y = 0.0, z = })
 \end{chapeloutput}
-there are no user-defined constructors for \chpl{C}, so \chpl{new} operators
-will invoke \chpl{C}'s compiler-generated constructor. The \chpl{x} argument
-of the compiler-generated constructor has the default value \chpl{0}.
+there are no user-defined initializers for \chpl{C}, so \chpl{new} expressions
+will invoke \chpl{C}'s compiler-generated initializer. The \chpl{x} argument
+of the compiler-generated initializer has the default value \chpl{0}.
 The \chpl{y} and \chpl{z} arguments have the default values \chpl{3.14} and
 \chpl{"Hello, World\!"}, respectively.
```

Comment:
I feel like these sentences should be "the default value(s) of ", but I might also be crazy . . .

Reply:
I think "the default value(s)" is OK, at least to my ear.